src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,261 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.math.BigDecimal;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static final char PLUS = '+';
+    private static final char MINUS = '-';
+    private static final char MULTIPLY = '*';
+    private static final char DIVIDE = '/';
+    private static final char POWER = '^';
+    private static Calculator calculator;
+    private static BigDecimal expected;
+    private static double positiveOperand;
+    private static double negativeOperand;
+    private static double zeroOperand;
+    private static double minDoubleValue;
+    private static double maxDoubleValue;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+        positiveOperand = 12.65;
+        negativeOperand = -132.228;
+        zeroOperand = 0;
+        minDoubleValue = Double.MIN_VALUE;
+        maxDoubleValue = Double.MAX_VALUE;
+    }
```

Comment:
```suggestion
    static void beforeAll() {
        calculator = new Calculator();
    }
```

let's remove these fields and just use plain numbers in tests
in test class they would not be treated as magic numbers